src/com/mambu/apisdk/util/RequestExecutorImpl.java

```diff
@@ -44,7 +46,7 @@
 
 	private URLHelper urlHelper;
 	private String encodedAuthorization;
-	private final static String UTF8_charset = HTTP.UTF_8;
+	private final static String UTF8_charset = StandardCharsets.UTF_8.name();
```

Comment:
Nice!  Please ignore my previous suggestion to do this exact update :)  sorry for rushing with the suggestion :)